rootfs/api/models/app.py

```diff
@@ -541,6 +542,7 @@ def verify_application_health(self, **kwargs):
             level=logging.DEBUG
         )
 
+    @backoff.on_exception(backoff.expo, ServiceUnavailable, max_tries=3)
```

Comment:
Yeah, I specifically decided not to use the `retrying` lib as it has not had a release since 2014 where as `backoff` was released a little bit ago. See https://github.com/deis/controller/issues/708